### PR TITLE
[4.1] Fix broken aria label

### PR DESCRIPTION
**Pull request by bembelimen**
Pull Request for Issue #36215 .

### Summary of Changes
Fix the wrong aria attribute for the search field in the filter section.

### Testing Instructions
Go to any search toolbar in the backend and inspect the search field.

### Actual result BEFORE applying this Pull Request
aria-describedby IS NOT the same as the ID of the description field.

### Expected result AFTER applying this Pull Request
aria-describedby IS the same as the ID of the description field.

### CODE REVIEW

**Comment by richard67** [comment]
Setting release blocker label as inherited from the issue.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 989452c527c2db3b540dfb354d776c4ef2645e8e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36225">issues.joomla.org/tracker/joomla-cms/36225</a>.</sub>

File: layouts/joomla/searchtools/default/bar.php [hunk 1]
```diff
@@ -35,7 +35,7 @@
 			<div class="input-group">
 				<?php echo $filters['filter_search']->input; ?>
 				<?php if ($filters['filter_search']->description) : ?>
-				<div role="tooltip" id="<?php echo $filters['filter_search']->name . '-desc'; ?>">
+				<div role="tooltip" id="<?php echo ($filters['filter_search']->id ?: $filters['filter_search']->name) . '-desc'; ?>">
```
Comment (laoneo) [inline layouts/joomla/searchtools/default/bar.php:38]: As this is a variable which can change, would it not be better to give the element a class which is always the same and reference that in CSS?

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 7345d78f7f5cfa3ed9ac5a855d7e9f87360b24c5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36225">issues.joomla.org/tracker/joomla-cms/36225</a>.</sub>

**Comment by richard67** [comment]
@bembelimen I'd set RTC because it has 2 tests, but if possible could you answer @laoneo 's review comment above? Thanks in advance.

**Comment by chmst** [comment]
@laoneo your suggestion is good. I will open a new issue for it

**Comment by richard67** [comment]
Ah I see Brian's test was before the last 2 commits, so no RTC yet.

@brianteeman Could you test again? Thanks in advance.

**Comment by laoneo** [comment]
Why a new issue, better to fix it from the beginning right?

**Comment by drmenzelit** [comment]
I have tested this item :white_check_mark: successfully on 0ab3d7ac237e72ffae0a40423f4c23d39a19f64e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36225">issues.joomla.org/tracker/joomla-cms/36225</a>.</sub>

**Comment by richard67** [comment]
I've added back @chmst 's test result since the 2 commits after that were just clean branch updates.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36225">issues.joomla.org/tracker/joomla-cms/36225</a>.</sub>

**Comment by richard67** [comment]
@bembelimen I've set RTC because it has 2 good tests, but maybe you want to check @laoneo 's latest comment.

**Comment by bembelimen** [comment]
Thx for the tests.

Yes I agree with @laoneo it could be a class. But for the sake of this PR which should just fisinx the wrong ID assignment I would like to see it in a new PR. So @chmst thanks for creating an issue.

**Pull request merged**